python/lsst/analysis/tools/actions/scalar/scalarActions.py

```diff
@@ -124,3 +124,36 @@ def __call__(self, data: KeyedData, **kwargs) -> Scalar:
             return 100.0 * result
         else:
             return result
+
+
+class MaxAction(ScalarAction):
+    vectorKey = Field[str]("Key of Vector to find maximum")
+
+    def getInputSchema(self) -> KeyedDataSchema:
+        return ((self.vectorKey, Vector),)
+
+    def __call__(self, data: KeyedData, **kwargs) -> Scalar:
+        mask = self.getMask(**kwargs)
+        return cast(Scalar, float(np.max(cast(Vector, data[self.vectorKey.format(**kwargs)])[mask])))
```

Comment:
Is the extra `float(...)` necessary here? Isn't that what `np.max()` returns?

Reply:
Yes it does return a float, but I noticed that in the other existing scalar actions the type is always specified so I was following that convention. np.max can also return a float if the array only contains ints, though, so I could leave it out and it would give either. However, since all of the other scalar actions seem to require a specific type I decided on a float.